models/dirwatcher.js

```diff
@@ -0,0 +1,59 @@
+import EventEmitter from "events";
+import fs from "fs";
+import { FILE_ADDED, FILE_MODIFIED, FILE_DELETED } from "./../config/constants";
+
+class DirWatcher extends EventEmitter {
+  constructor() {
+    super();
+    this.fileStats = {};
+    this.fileStatsTemp = {};
+    this.timerInterval;
+  }
+  checkIndividualFileStatus = file => {
+    const fileLastModifiedTimeStamp = this.fileStats[file];
+    const fileOriginalTimeStamp = this.fileStatsTemp[file];
+    if (!fileLastModifiedTimeStamp) {
+      return { [file]: FILE_ADDED };
+    } else if (!fileOriginalTimeStamp) {
+      return { [file]: FILE_DELETED };
```

Comment:
Please, recheck these method. Nothing is logged in console in case of file deletion.